src/java/test/org/apache/zookeeper/test/LoadFromLogTest.java

```diff
@@ -69,25 +54,42 @@
     private static final int TRANSACTION_OVERHEAD = 2;
     private static final int TOTAL_TRANSACTIONS = NUM_MESSAGES + TRANSACTION_OVERHEAD;
 
-    /**
-     * test that all transactions from the Log are loaded, and only once
-     * @throws Exception an exception might be thrown here
-     */
-    @Test
-    public void testLoad() throws Exception {
-        final String hostPort = HOST + PortAssignment.unique();
+    private String hostPort;
+    private int port;
+    private ZooKeeper zk;
+    private ServerCnxnFactory serverCnxnFactory;
+    private File tmpDir;
+    private ZooKeeperServer zks;
+
+    @Before
+    public void setUp() throws Exception {
+        hostPort = HOST + PortAssignment.unique();
```

Comment:
While you're in here can you also fix this? We should be using this value from ClientBase. Is there a reason why we can't convert this class over to subclass ClientBase?